driver.go

```diff
@@ -48,13 +48,33 @@ func (l *lvmDriver) Create(req volume.Request) volume.Response {
 		return resp(err)
 	}
 
-	cmdArgs := []string{"-n", req.Name}
+	cmdArgs := []string{"-n", req.Name, "--setactivationskip", "n"}
+	snap, ok := req.Options["snapshot"]
+	isSnapshot := ok && snap != ""
 	s, ok := req.Options["size"]
-	if !ok || (ok && s == "") {
+	hasSize := ok && s != ""
+
+	// no size is valid for thinly-provisioned snapshot volumes
+	if !hasSize && !isSnapshot {
```

Comment:
@cschauer If I am trying to create a snapshot of a regular volume {foobar} (where size is mandatory) and I forgot to pass the size.
e.g. `docker volume create -d lvm --name foobar_snapshot --opt snapshot=foobar`. Code should not pass this check, and should error out at this point saying "Please specify a size with --size"

Right now, it is passing this check and blowing up later. I am still trying to think how to solve this problem as one would need to know if it is a snapshot of a regular volume VS a thin volume (where size is optional).

Shishir